### DISCUSSION TITLE
chapter_divide_and_conquer/summary/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#619] [Announcements]
# chapter_divide_and_conquer/summary/

动画图解、一键运行的数据结构与算法教程

https://www.hello-algo.com/chapter_divide_and_conquer/summary/

**Comment by Magicperformer** [comment]
终于快看完了，叉会儿腰，谢谢K神的分享

**Comment by HuaWang135608** [comment]
Over

**Comment by ztkuaikuai** [comment]
❤

**Comment by weiiiyan** [comment]
“子问题能否合并”、治（合并），这些描述总感觉不对，分是划分问题，将大的问题分成小点子问题；治是解决子问题（当然子问题还会依赖更小的孙子问题，所以通常会递归到最小子问题）；而除分、治以外，还有个以子问题的解得到父问题的解的过程，可以是合并，也可以子问题的解就算父问题的解，但这个过程不会是属于”治“。即治不包含合并、“子问题能否合并”改为”子问题的解是否能推导出父问题的解“。
